workflow/__init__.py

```
@@ -15,7 +15,9 @@
 
 
 def get_test_dir(bucket_name):
-    return bucket_name.removeprefix(kTestBucket + ":")
+    if not bucket_name.startswith(kTestBucket+":"):
+        raise
```

Should we raise a particular error?

---

done!